src/v2/Apps/Order/Components/ArtworkSummaryItem.tsx

```diff
@@ -31,9 +31,14 @@ const ArtworkSummaryItem: React.FC<ArtworkSummaryItemProps> = ({
   },
   ...others
 }) => {
-  const artwork = get({}, () => lineItems?.edges?.[0]?.node?.artwork)
+  const firstLineItem = get({}, () => lineItems?.edges?.[0]?.node!)
+  const { artwork, artworkVersion } = firstLineItem!
 
-  const { artistNames, title, date, shippingOrigin, image } = artwork!
+  if (!artwork) console.warn("artwork missing")
+  if (!artworkVersion) console.warn("artwork version missing")
+
+  const { artistNames, title, image } = artworkVersion || {}
+  const { date, shippingOrigin } = artwork || {}
```

Comment:
Looks like there is information that's not available on artwork version. I think a good first step is to not 500 the entire page. 👍